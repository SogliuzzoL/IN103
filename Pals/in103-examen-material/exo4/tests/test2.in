78
86
11
92
82
2
29
12
3
81
