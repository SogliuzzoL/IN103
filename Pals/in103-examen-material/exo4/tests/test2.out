-1
13416
7566
8770
8770
7934
7934
7934
7918
7918
